gc/base/standard/WorkPacketsSATB.hpp

```diff
@@ -42,13 +42,25 @@ class MM_WorkPacketsSATB : public MM_WorkPackets
 	virtual bool initialize(MM_EnvironmentBase *env);
 	virtual void tearDown(MM_EnvironmentBase *env);
 
-	MM_IncrementalOverflow *getIncrementalOverflowHandler() const { return (MM_IncrementalOverflow*)_overflowHandler; }
+	MM_IncrementalOverflow *getIncrementalOverflowHandler() const
+	{
+		return ((MM_IncrementalOverflow *) _overflowHandler);
```

Comment:
The below space can be removed.
```suggestion
		return ((MM_IncrementalOverflow *)_overflowHandler);
```